generics/messageConstants/common.js

```diff
@@ -87,4 +87,9 @@ module.exports = {
   PRIVATE: 'private',
 	WRITE_PERMISSION: 'w',
 	READ_PERMISSION: 'r',
+  pdf_report_generated : "Pdf report generated successfully",
```

Comment:
@borkarsaish65 there are api response messages right? should we have them in common.js?

Reply:
removed